types/relay-runtime/lib/util/RelayRuntimeTypes.d.ts

```diff
@@ -40,3 +40,10 @@ export interface CacheConfig {
     metadata?: { [key: string]: unknown };
     transactionId?: string | null;
 }
+
```

Comment:
Technically this is also being done @ https://github.com/DefinitelyTyped/DefinitelyTyped/pull/48638/files#diff-1d1c1a7f02cdc485daf916e9e6803e55f23b46d3243687637c1441c2c34114b0R50-R52 which has further reaching usages. But think its still valid to include with this pr.